core/base/src/TStorage.cxx

```diff
@@ -200,8 +200,10 @@ void *TStorage::ReAlloc(void *ovp, size_t size)
 #else
    void *vp = ::operator new(size);
 #endif
-   if (vp == nullptr)
+   if (vp == nullptr) {
       Fatal(where, "%s", gSpaceErr);
```

Comment:
Note: Fatal can not return.

Reply:
Should I revert this change? Or I leave it as now even if it is redundant?

Reply:
The 'best' would be to mark it as unreacheable, but this requires a little bit of infrastructure (see for example the usage in Celeritas https://github.com/celeritas-project/celeritas/blob/802fb37d6f2720311a4000c38581e712d6096045/src/corecel/Macros.hh#L101)
For now maybe just:
 ```suggestion
      Fatal(where, "%s", gSpaceErr);
      return nullptr; // Unreacheable.
```